packages/amplify_analytics_plugin_interface/lib/src/Analytics/AnalyticsUserProfile.dart

```diff
@@ -13,19 +13,21 @@
  * permissions and limitations under the License.
  */
 
-import '../../analytics_plugin_interface.dart';
+import '../../amplify_analytics_plugin_interface.dart';
 
 class AnalyticsUserProfile {
-
   String name;
   String email;
   String plan;
   AnalyticsUserProfileLocation location;
   AnalyticsProperties properties;
 
-  AnalyticsUserProfile({
-    String name, String email, String plan, AnalyticsUserProfileLocation location, AnalyticsProperties analyticsProperties
-  }) {
+  AnalyticsUserProfile(
+      {String name,
```

Comment:
A trivial suggestion to keep the params at the same column!
```suggestion
  AnalyticsUserProfile({
      String name,
```